version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.20.2-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.20.2"
```

Comment:
Are we going to release now?

Reply:
@bilki I'm holding back from this, so we can decide the release timeline appropiatedly in a separate PR.